server/src/test/java/org/elasticsearch/repositories/RepositoriesServiceTests.java

```diff
@@ -202,9 +202,7 @@ public RepositoryMetadata getMetadata() {
         }
 
         @Override
-        public SnapshotInfo getSnapshotInfo(SnapshotId snapshotId) {
-            return null;
-        }
+        public void getSnapshotInfo(GetSnapshotInfoContext context) {}
```

Comment:
Doing nothing here leaks the context. Should we complete it (maybe with a UOE) or assert that this method isn't called?

Reply:
++ made it throw